spec/activerecord-bitemporal/bitemporal_scope_spec.rb

```diff
@@ -777,6 +777,93 @@ class Article < ActiveRecord::Base
       end
     end
 
+    describe "ActiveRecord::Bitemporal.transaction_at" do
+      let(:transaction_datetime) { "2019/05/05".in_time_zone }
+      let(:relation) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime) { User.all } }
+      it { is_expected.to have_valid_at(time_current, table: "users") }
+      it { is_expected.to have_transaction_at(transaction_datetime, table: "users") }
+
+      context "transaction_datetime is nil" do
+        let(:transaction_datetime) { nil }
+        it { is_expected.to have_bitemporal_at(time_current, table: "users") }
+      end
+
+      describe ".transaction_at" do
+        let(:user_transaction_datetime) { "2019/05/05".in_time_zone }
+        let(:relation) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime) { User.transaction_at(user_transaction_datetime) } }
+        it { is_expected.to have_valid_at(time_current, table: "users") }
+        it { is_expected.to have_transaction_at(user_transaction_datetime, table: "users") }
+      end
+
+      context "with ActiveRecord::Bitemporal.ignore_transaction_datetime" do
+        let(:relation) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime) { ActiveRecord::Bitemporal.ignore_transaction_datetime { User.all } } }
+        it { is_expected.to have_valid_at(time_current, table: "users") }
+        it { is_expected.to not_have_transaction_at(table: "users") }
+      end
+
+      context ".ignore_transaction_datetime" do
+        let(:relation) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime) { User.ignore_transaction_datetime } }
+        it { is_expected.to have_valid_at(time_current, table: "users") }
+        it { is_expected.to not_have_transaction_at(table: "users") }
+      end
+
+      describe "ActiveRecord::Bitemporal.transaction_at!" do
+        let(:transaction_datetime2) { "2019/06/06".in_time_zone }
+        let(:relation) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime2) { User.all } } }
+        it { is_expected.to have_valid_at(time_current, table: "users") }
+        it { is_expected.to have_transaction_at(transaction_datetime2, table: "users") }
+      end
+    end
+
+    describe "ActiveRecord::Bitemporal.transaction_at!" do
+      let(:transaction_datetime) { "2019/05/05".in_time_zone }
+      let(:relation) { ActiveRecord::Bitemporal.transaction_at!(transaction_datetime) { User.all } }
+      it { is_expected.to have_valid_at(time_current, table: "users") }
+      it { is_expected.to have_transaction_at(transaction_datetime, table: "users") }
+
+      context "transaction_datetime is nil" do
+        let(:transaction_datetime) { nil }
+        it { is_expected.to have_bitemporal_at(time_current, table: "users") }
+
+        context "with .transaction_at" do
+          it { is_expected.to have_bitemporal_at(time_current, table: "users") }
```

Comment:
Is this the same as the above test case?
https://github.com/kufu/activerecord-bitemporal/blob/5181ec36bb0839fc60ec3075e4e7348527b6f5e7/spec/activerecord-bitemporal/bitemporal_scope_spec.rb#L826

Reply:
Thanks! Fix it.
https://github.com/kufu/activerecord-bitemporal/pull/82/commits/839a5a5debf09a9810c91238679aa884711de24c